src/pages/icas/CheckICA.tsx

```diff
@@ -469,7 +474,9 @@ export default class CheckIca extends React.Component<IProps, IState> {
         this.initState();
 
         const reader = new FileReader();
-        reader.readAsText(event.target.files[0]);
+        const file = event.target.files[0];
```

Comment:
in to `checkXML` function on `if (result.right.status === 200) {` i see
```
CheckICA.tsx:475 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'right')
    at CheckICA.tsx:475
```

Reply:
Solved